py_mmd_tools/nc_to_mmd.py

```diff
@@ -1155,13 +1155,13 @@ def to_mmd(self, collection=None, checksum_calculation=False, mmd_yaml=None,
 
         mmd_docs = 'https://htmlpreview.github.io/?https://github.com/metno/mmd/blob/master/' \
                    'doc/mmd-specification.html#collection-keywords'
-        default_collections = ['ADC', 'METNCS']
+        default_collection = 'METNCS'
         if collection is None:
```

Comment:
```suggestion
        if collection == '':
```

Reply:
We already know (Line 1136) if collection is given.

Reply:
(not None)

Reply:
no, we check in line 1136 if it is `none` or `str` but we don't change it. Unless it is given, it will still be `None`. But we could check if someone has provided an empty string...

Reply:
If None or not str: 
            raise ValueError('collection must be of type str or list')

Reply:
no, we check in line 1136 if it is none or str but we don't change it. Unless it is given, it will still be None. But we could check if someone has provided an empty string...